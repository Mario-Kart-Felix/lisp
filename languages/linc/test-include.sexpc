
(include "foo.h")
(include <foo.h>)
(include "foo.h" "bar.h" <baz.h>)

(include)
(include "foo\".h")
(include <foo>bar.h>)
(include "good1.h" "foo\".h" "good2.h")
(include <good1.h> <foo>bar.h> <good2.h>)

