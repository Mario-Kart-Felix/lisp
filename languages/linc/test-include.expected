/* ------------------------- DO NOT EDIT! --------------------------------- */
/* WARNING: This file is generated automatically by LINC from the source    */
/* file test-include.sexpc                                                  */
/* ------------------------- DO NOT EDIT! --------------------------------- */

#include "foo.h"
#include <foo.h>
#include "foo.h"
#include "bar.h"
#include <baz.h>
